### ISSUE TITLE
[Suggestion] highwaybuilder improvements

### ISSUE THREAD

**Issue by playeur242** [MeteorDevelopment/meteor-client#1908]
### Describe the feature

welp here what would be great to add to highways builder :

- a basic delay for breaking and putting blocks because it just make tone of fake blocks if you have high ping.

-a Y check because sometimes you fall and it make highway under the highways so it could be great to have a auto stop when you fall or something

-a safewalk working with highwaybuilder because safewalk is alrdy binded with scaffold and sometimes you fall because of fakeblocks

well ye i think that's all thx to whoever actually readed it ._.

### Before submitting a suggestion

- [X] This feature doesn't already exist in the client.
(I have checked every module and their settings on the **latest dev build**)

- [X] This wasn't already suggested.
(I have checked every suggestion on github)

- [X] This feature is a valid suggestion.
(I have read the FAQs and Guides)

**Comment by CadaverThrasher** [comment]
Yes, sometimes I fall a Y level for various reasons (a mob for example) and Highway Builder just continues on the new y level as if nothing happened.
